ykllvmwrap/src/jitmodbuilder.cc

```diff
@@ -296,23 +307,24 @@ class JITModBuilder {
     GlobalMappings.insert({CF, FAddr});
   }
 
-  // Store a vector and its length as a struct. This allows us to save
-  // arguments when passing this information via the deoptimisation call.
-  AllocaInst *storeVecWithLength(IRBuilder<> &Builder, Value *Vec, Value *Len) {
+  // Generate LLVM IR to create a struct and store the given values.
```

Comment:
Does this store into the heap allocation?

Reply:
It's an alloca, which is stack allocated.